packages/components/src/DateTimePickers/pickers/MonthPicker/MonthPicker.component.js

```diff
@@ -54,6 +54,7 @@ class MonthPicker extends React.PureComponent {
 								return (
 									<td key={index} {...tdProps}>
 										<button
+											name={index}
```

Comment:
Why did you added this name attribute in month picker ?

Reply:
for the navigation by keyboard.
function getAllItems(calendarRef) {
	return calendarRef.querySelectorAll('td > button[name]');
}
and as your previous comment, I think about use 'data-valid' to replace 'name'

Reply:
changed to be 'data-value'